modules/log6x-collection-setup.adoc

```diff
@@ -0,0 +1,201 @@
+----
+
+=== Writing audit logs
+The write-audit-logs-clusterrole.yaml file defines a ClusterRole that grants permissions to create audit logs in the Loki logging system.
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-logging-write-audit-logs
+rules:                                              <1>
+  - apiGroups:                                      <2>
+      - loki.grafana.com                            <3>
+    resources:                                      <4>
+      - audit                                       <5>
+    resourceNames:                                  <6>
+      - logs                                        <7>
+    verbs:                                          <8>
+      - create                                      <9>
+----
+<1> rules: Defines the permissions granted by this ClusterRole.
+<2> apiGroups: Specifies the API group loki.grafana.com.
+<3> loki.grafana.com: The API group responsible for Loki logging resources.
+<4> resources: Refers to the resource type this role manages, in this case, audit.
+<5> audit: Specifies that the role manages audit logs within Loki.
+<6> resourceNames: Defines the specific resources that the role can access.
+<7> logs: Refers to the logs that can be managed under this role.
+<8> verbs: The actions allowed on the resources.
+<9> create: Grants permission to create new audit logs.
+
+=== Writing infrastructure logs
+The write-infrastructure-logs-clusterrole.yaml file defines a ClusterRole that grants permission to create infrastructure logs in the Loki logging system.
+
+.Sample YAML
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-logging-write-infrastructure-logs
+rules:                                              <1>
+  - apiGroups:                                      <2>
+      - loki.grafana.com                            <3>
+    resources:                                      <4>
+      - infrastructure                              <5>
+    resourceNames:                                  <6>
+      - logs                                        <7>
+    verbs:                                          <8>
+      - create                                      <9>
+----
+<1> rules: Specifies the permissions this ClusterRole grants.
+<2> apiGroups: Specifies the API group for Loki-related resources.
+<3> loki.grafana.com: The API group managing the Loki logging system.
+<4> resources: Defines the resource type that this role can interact with.
+<5> infrastructure: Refers to infrastructure-related resources that this role manages.
+<6> resourceNames: Specifies the names of resources this role can manage.
+<7> logs: Refers to the log resources related to infrastructure.
+<8> verbs: The actions permitted by this role.
+<9> create: Grants permission to create infrastructure logs in the Loki system.
+
+=== ClusterLogForwarder editor role
```

Comment:
Similar to all our other CRDs there is a role provided by the tooling already that does this: `clusterlogforwarders.observability.openshift.io-v1-edit` (also `-admin`, `-crdview` and `-view`).